content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+dependencies.
+
+Visit our [virtual services reference documentation](/docs/reference/config/istio.networking.v1alpha3/#VirtualService) to review all the enabled keys and values.
+
+## Gateways
+
+The Istio gateways are layer 7 load balancers. With an Istio gateway, you can
+configure routes for HTTP and TCP traffic regardless of where the gateway runs.
+Your mesh can have any number of gateways and multiple different gateway
+implementations can co-exist within your mesh. Using a gateway, you can
+configure workload labels to reuse your existing network appliances such as:
+firewall functions, caching, authentication, network address translation, and
+IP address management.
+
+By separating the layer 4 and 6 specifications from the layer 7, the Istio
+gateways overcome the shortcomings of the layer 4 and 6 specifications. Istio
+only configures the functions that layer 7 load balancing proxies implement
+uniformly. To use standard Istio rules to control HTTP requests as well as TCP
+traffic entering a gateway, you must bind a virtual service to the gateway. The
+following example shows a possible gateway configuration to ingress external
+HTTPS traffic to the mesh:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: ext-host-gwy
+spec:
+  servers:
+  - port:
+      number: 443
+      name: https
+      protocol: HTTPS
+    hosts:
+    - ext-host
+    tls:
+      mode: SIMPLE
+      serverCertificate: /tmp/tls.crt
+      privateKey: /tmp/tls.key
+{{< /text >}}
+
+To configure the external HTTPS traffic to flow from the `ext-host` host, you
+must add the Istio gateway to a Istio virtual service as shown:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: virtual-svc
+spec:
+  hosts:
+    - ext-svc
+  gateways:
+    - ext-host-gwy
+{{< /text >}}
+
+The following diagram shows the configured routes for the external HTTPS
+traffic coming from the external host on port 443 to the `virtual-svc` virtual
+service through the `ext-host-gwy` gateway to reach the `my-svc` service
+within the mesh:
+
+{{< image width="50%"
+    link="./net-config-6.svg"
+    caption="Configurable routes for the external HTTPS traffic"
+    >}}
+
+Visit our [gateways reference documentation](/docs/reference/config/istio.networking.v1alpha3/#Gateway) to review all the enabled keys and values.
+
+### Ingress gateways
+
+Use ingress gateways to configure ingress traffic routes into the service mesh.
+You can add as many gateways as you need. For example, you can have one gateway
+for private traffic and another for public traffic. Thus, you balance the
+different loads independently from each other.
+
+### Egress gateways
+
+Use egress gateways to configure an exit route for traffic leaving the mesh.
+You can configure each egress gateway to use its own policies and telemetry.
+You can use egress gateways if only particular application services can or
```

Comment:
Isn't there a 'must' here? That is, you _must_ use egress gateways if you want to permit traffic off the mesh?

Reply:
no, sending egress traffic via a gateway is optional.

Reply:
ack.